sample_code_submission/model.py

```diff
@@ -1,26 +1,25 @@
 import os
 from sys import path
 import numpy as np
-import pandas as pd
-from math import sqrt, log
-from xgboost import XGBClassifier
-from sklearn.utils import shuffle
+
 from sklearn.metrics import roc_auc_score
 from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import StandardScaler
-from sklearn.multioutput import MultiOutputRegressor
 import matplotlib.pyplot as plt
 from tqdm import tqdm
-import seaborn as sns
+import pickle
+import json
 
+# import mplhep as hep
 
+# hep.set_style("ATLAS")
 # ------------------------------
 # Absolute path to submission dir
 # ------------------------------
 submissions_dir = os.path.dirname(os.path.abspath(__file__))
 path.append(submissions_dir)
 
-from systematics import postprocess
+from systematics import postprocess, DER_data
```

Comment:
why do you have this import here. We only supply Systematics in the constructor

Reply:
actually its useless, I used to have an meaning. I will fix it